test_run.lua

```diff
@@ -284,6 +285,26 @@ local function grep_log(self, node, what, bytes, opts)
     return found
 end
 
+-- Checks fn with delay until timeout or fn returns true, otherwise returns false
+local function wait_cond(self, fn, timeout, delay)
+    assert(timeout ~= nil)
+    delay = delay or 0.001
+    local t1 = clock.monotonic()
```

Comment:
Don't sure about monotonic, but monotonic64 is fast enough, because does not do timezone-related work. Can you check it or change?

Reply:
The `monotonic` and `monotonic64` uses same method to get time:
```
clock_gettime(CLOCK_MONOTONIC, &ts);
```

Reply:
Thanks!